src/ontology/mondo-edit.obo

```diff
@@ -419921,11 +419924,15 @@ xref: Orphanet:314655 {source="MONDO:equivalentTo"}
 xref: SCTID:768555009 {source="MONDO:equivalentTo"}
 xref: UMLS:C4708510 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1636705"}
 is_a: MONDO:0002254 {source="https://orcid.org/0000-0001-5208-3432"} ! syndromic disease
+is_a: MONDO:0015159 {source="https://orcid.org/0000-0001-9310-0163"} ! multiple congenital anomalies/dysmorphic syndrome-intellectual disability
 is_a: MONDO:0016904 {source="Orphanet:314655"} ! partial deletion of the long arm of chromosome 5
+is_a: MONDO:1060108 {source="https://orcid.org/0000-0001-9310-0163"} ! PURA-related severe neonatal hypotonia-seizures-encephalopathy syndrome
```

Comment:
Based on [here](https://www.ebi.ac.uk/ols4/ontologies/ordo/classes/http%253A%252F%252Fwww.orpha.net%252FORDO%252FOrphanet_314655), Please add the Orphanet x-ref as a source